### ISSUE TITLE
Not proper navigation when switching from key selection dropdown to the next item in Instrument list

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#20657]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Bug description

When there are multiple instruments with “key drop-downs” in the search, if you press down, the **focus will move to the next key drop-down instead of the next item in the list.**

### Steps to reproduce

https://github.com/musescore/MuseScore/assets/90187801/82c7056c-47f4-4803-9689-64dffcaec558

### MuseScore Version

4.3

### Regression

No.

### Operating system

Windows 10

**Comment by bkunda** [comment]
I think in this case, there's no practical reason why you'd want to navigate between the key selector dropdowns. I'd expect that, if a key selector dropdown was in focus, pressing **down** would shift focus to the next instrument in the list (not the next dropdown). 
@shoogle would you agree with this?

**Comment by shoogle** [comment]
I agree, moving to the next instrument is probably more intuitive than moving to the next dropdown.

However, in other applications, focus would likely stay on the current dropdown when you press the Down key, and either the dropdown would open so you can see all the values, or it would remain closed and the value would change (e.g. from 'B' to 'C').

So I wouldn't want to rush into making a decision based on this case alone. Instead, I'm putting together an accessibility document to harmonize keyboard navigation between MuseScore and Audacity. This is certainly an interesting case to consider in that design.

**Comment by shubham-shinde-442** [comment]
Can I create a pull request for this issue ?

**Comment by bkunda** [comment]
Postponing this on @shoogle's recommendation. Seems like it needs some more thought, and we've run out of time to address it for 4.4.
